dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,9 @@
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+mpTelemetryLogging=MicroProfile Telemetry 2.0
+mpTelemetryLogging.desc=MicroProfile Telemetry Logging gathers data from various sources and forwards the data to the configured MicroProfile Telemetry Log Exporter.
+
+source=Source
+source.desc=Specifies a list of comma-separated sources that are used by MicroProfile Telemetry. Valid values are message, trace, ffdc.
```

Comment:
Specifies a list of comma-separated sources for the MicroProfile Telemetry. Valid values for the source attribute are message, trace, and ffdc.

Reply:
Fixed.